src/app/guide/pages/data-submission-guide/data-submission-guide.component.ts

```diff
@@ -0,0 +1,62 @@
+import {Component, OnInit} from '@angular/core';
+import {AlertService} from "@shared/services/alert.service";
+import { Converter } from 'showdown';
+import {ActivatedRoute, Router} from '@angular/router';
+
+const BASE_MARKDOWN_DIR = '/assets/guide-markdown/'
+
+enum ROUTES {
+  DataRequirements = 'data-requirements',
+  StepByStep = 'step-by-step',
+  AfterSubmission = 'after-submission'
+}
+
+const MARKDOWN_FILES_AND_ROUTES = {
+  [ROUTES.DataRequirements]: [BASE_MARKDOWN_DIR + 'data-requirements.md', 'Data requirements'],
```

Comment:
Have you considered setting the heading from the file name unless a heading is provided explicitly? 
I.e. `data-requirements.md` -> `Data Requirements`?

Reply:
I thought about that but I prefer explicit things in general. I don't know exactly what gabs wants to call everything right now either

Reply:
I would prefer a simple default behaviour and the ability to break the standard format when needed. Otherwise we accumulate a lot of duplicate code.